sdks/java/core/src/main/java/org/apache/beam/sdk/coders/VoidCoder.java

```diff
@@ -38,7 +39,10 @@ public static VoidCoder of() {
   private VoidCoder() {}
 
   @Override
-  public void encode(Void value, OutputStream outStream) {
+  public void encode(Void value, OutputStream outStream) throws IOException {
```

Comment:
It turns out that we don't need this change since the case to the Void type should never work from an arbitrary user type. I remove this change and merged your code.